src/web/wordpress_api_calls.py

```diff
@@ -0,0 +1,56 @@
+import requests
+import re
+import json
+from django.conf import settings
+from datetime import datetime, timedelta
+
+
+def get_wp_credentials():
+    regex = r"var\s+wpAmeliaNonce\s*=\s*['\"]?([a-fA-F0-9]+)['\"]?"
+
+    session = requests.Session()
+    headers1 = {"Cookie": "wordpress_test_cookie=WP Cookie check"}
+    datas = {
+        "log": settings.WP_USER,
+        "pwd": settings.WP_PASSWORD,
+        "wp-submit": "Log In",
+        "redirect_to": settings.WP_ADMIN,
+        "testcookie": "1",
+    }
+    session.post(settings.WP_LOGIN, headers=headers1, data=datas)
+    resp = session.get(settings.WP_ADMIN)
+
+    match = re.search(regex, resp.text)
+    if not match:
+        print("No nonce found")
+        return
+    nonce = match.group(1)
+
+    return nonce, session
+
+
+def get_massage_customer_data_from_wp(day_past, day_future, nonce, session):
+    date_sync_before = (datetime.today() - timedelta(days=day_past)).strftime(
+        "%Y-%m-%d"
+    )
+    date_sync_week = (datetime.today() + timedelta(days=day_future)).strftime(
+        "%Y-%m-%d"
+    )
+
+    url = settings.WP_URL_APPOINTMENTS.format(
+        nonce=nonce,
+        date_sync_before=date_sync_before,
+        date_sync_week=date_sync_week,
+    )
+    wp_json = session.get(url).text
+    wp_massage_customer = json.loads(wp_json)
```

Comment:
```suggestion
    wp_massage_customer = session.get(url).data()
```

Reply:
From where do you get data() ?
should it not be: `wp_massage_customer = json.loads(session.get(url).text)` ?